add additional whitelisting in .gitignore

as3mbus opened the pull request:
i am proposing this change to assist the installation process in project scope. 

especially when user trying to add ue git plugin inside the project to ease distribution.  since recommended `.gitignore` of pbcore ignores git-lfs.exe files which required by the plugin to works

as3mbus committed:
add additional whitelisting in .gitignore

i am proposing this change to assist the installation process in project scope. 

especially when user trying to add ue git plugin inside the project to ease distribution. 
since recommended `.gitignore` of pbcore ignores git-lfs.exe files which required by the plugin to works

mastercoms reviewed (changes requested):
Thanks, great idea! Could you add it for the other executables as well?

as3mbus commented:
the other executable doesn't get ignored by pbcore .gitignore by default. or you'd say just add it to gitignore whitelist regardless the condition ?

mastercoms commented:
Some people use the Epic gitignore which ignores everything by default

as3mbus committed:
Add additional white listing for executables

Add additional .gitignore white listing for executable required by the plugin to works

as3mbus commented:
updated with all executables files
